paddle/fluid/operators/split_op.cc

```diff
@@ -78,9 +78,18 @@ class SplitOp : public framework::OperatorWithKernel {
 
 #ifdef PADDLE_WITH_MKLDNN
     if (this->CanMKLDNNBeUsed(ctx, input_data_type)) {
-      return framework::OpKernelType(input_data_type, ctx.GetPlace(),
-                                     framework::DataLayout::kMKLDNN,
-                                     framework::LibraryType::kMKLDNN);
+      // OneDNN uses blocking format, which cannot be always
```

Comment:
As you describe in teams, oneDNN split only work well when avx512 instructions available, in this way why not check if avx512 is supported in the condition together with this->CanMKLDNNBeused ?

Reply:
Currently under avx512 instructions, what ops are using block format ? If under avx512 there are still ops using block format, maybe this way is good.

Reply:
On avx512 blocking format may be chosen later, that way it will still be working